WooCommerce/Classes/ViewRelated/Fancy Alerts/FancyAlertViewController+Payments.swift

```diff
@@ -0,0 +1,145 @@
+import Foundation
+import WordPressUI
+
+/// We are going to need to write a new modal view to present user facing messages related to payments.
+/// https://github.com/woocommerce/woocommerce-ios/issues/3980
+/// In the meantime, this, based on FancyAlertViewController, will have to do.
+/// This extension will probably be removed when 3980 is implemented
+public extension FancyAlertViewController {
```

Comment:
This kind of gets the job done for this iteration. It provides visual, tangible feedback on the UI, and validates the implementation of the underlying layers.

The UI does not match the designs, but we will take care of that in a later iteration.